src/server-code/admin-server.js

```diff
@@ -189,13 +189,14 @@ export default function runAdminServer(serverFalcorModel) {
       });
     }
   });
-
+  app.get('/robots.txt', (req, res) => {
+    res.send(robotsTxt);
```

Comment:
If you just make this `res.status(200).send(robotsTxt)` that'd be great, I think that's probably the default, but it's better to be specific (200 is the HTTP Status Code for "all is good" by the way if you didn't know :)) https://en.wikipedia.org/wiki/List_of_HTTP_status_codes

Reply:
And, the reason it wasn't showing up nice is because we didn't supply the `Content-Type` header, so on top of the `status` method we can also use the `type` method and do as follows: `res.status(200).type('txt').send(robotsTxt)`